tsconfig.json

```
@@ -5,21 +5,14 @@
 		"outDir": "build",
 		"sourceMap": true,
 		"declaration": true,
-		"target": "es6",
+		"target": "es2017",
```

This could be updated to
```suggestion
		"target": "es2018",
```
with dropped node 8 support, the only thing node 10 doesn't support is `Proxy "ownKeys" handler, duplicate keys for non-extensible targets`: https://node.green/#ES2018

---

updated